README.md

```diff
@@ -147,7 +147,11 @@ make e2e
 
 ## Configuration
 
-See [examples/config.yaml](./examples/config.yaml).
+There is a default configuration in [examples/config.yaml](./examples/config.yaml).
+
+Remember to open this configuration file, modify all **DO_CHANGE_ME** in it to your own.
+
+Pay attention to the meaning of each item to ensure that it is you want.
```

Comment:
"that it is you want" -> "that it is what you want"

I highly recommend the usage of Grammarly for documentation update.